pages/components/input/input.html

```diff
@@ -0,0 +1,206 @@
+            class="drawer-lists flex flexAlignItemsCenter"
+            href="../button/button.html"
+          >
+            <p class="drawer-list-content">Buttons</p>
+          </a>
+          <a
+            class="drawer-lists flex flexAlignItemsCenter"
+            href="../card/card.html"
+          >
+            <p class="drawer-list-content">Cards</p>
+          </a>
+          <a
+            class="drawer-lists flex flexAlignItemsCenter"
+            href="../image/image.html"
+          >
+            <p class="drawer-list-content">Image</p>
+          </a>
+          <a
+            class="drawer-lists flex flexAlignItemsCenter"
+            href="../input/input.html"
+          >
+            <p class="drawer-list-content">Input</p>
+          </a>
+        </div>
+      </aside>
+      <main class="main-content m-5 width-90">
+        <h1 class="mb-4">Inputs</h1>
+        <p class="mb-4">
+          The Input component are used to get user input in a text field.
+        </p>
+        <div class="text-input-wrapper">
+          <h2 class="mt-4 mb-4">Text Input</h2>
+          <div class="input-wrapper flex flexCol p-4">
+            <label for="text-input" class="mb-3">Name:</label>
+            <input
+              type="text"
+              class="text-input"
+              name="name"
+              placeholder="Enter the name"
+            />
+          </div>
+          <p class="mt-4 mb-4">
+            The Input component are used to get user input in a text field. For
+            using these inputs in your codebase add <b>'text-input'</b> class in
+            your input element and for the addition of paddings and margins, you
+            can use their utility classes.
+          </p>
+          <div class="code-wrapper">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28195%2C216%2C236%2C1%29&t=panda-syntax&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522input-wrapper%2520flex%2520flexCol%2520p-4%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522text-input%2522%2520class%253D%2522mb-3%2522%253EName%253A%253C%252Flabel%253E%250A%2520%2520%2520%2520%2520%2520%253Cinput%2520type%253D%2522text%2522%2520class%253D%2522text-input%2522%2520name%253D%2522name%2522%2520placeholder%253D%2522Enter%2520the%2520name%2522%2520%252F%253E%250A%253C%252Fdiv%253E"
+              style="
+                width: 80%;
+                height: 131px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+        </div>
+        <div class="validation-input-wrapper">
+          <h2 class="mt-4 mb-4">Validation Input</h2>
+          <div class="input-wrapper flex flexCol p-4">
```

Comment:
Not sure if using camel case for one of the classes (`flexCol`) is a good idea where hyphenated words have been used all around. Should we use `flex-col` instead?

Reply:
I am using this way only for layout utilities. So, as to make a difference from other classes.